bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BRecordType.java

```diff
@@ -61,8 +61,9 @@ public class BRecordType extends BStructureType implements RecordType {
      * @param sealed flag indicating the sealed status
      * @param typeFlags flags associated with the type
      */
-    public BRecordType(String typeName, Module pkg, long flags, boolean sealed, int typeFlags) {
+    public BRecordType(String typeName, String internalName, Module pkg, long flags, boolean sealed, int typeFlags) {
```

Comment:
should we remove other constructor where internal name can be null? or we can set internal name = typeName and add only declaration in the class private String internalName ;

Reply:
Updated the constructor to use the `typeName`